data/samMintah.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "Samuel Mintah",
+  "type": "personal",
+  "bio": "Software Engineer, Tech junkie, early adopter, and futurist. #AI #ML #gadgetlove #innovation",
```

Comment:
If #AI #ML ... are tags that you want to add then you can add them separately.
See this, https://linkfree.eddiehub.io/docs/how-to-guides/tags
If it's part of the bio then it looks good.